database/src/main/scala/cromwell/database/obj/Metadatum.scala

```diff
@@ -10,6 +10,7 @@ case class Metadatum
   index: Option[Int],
   attempt: Option[Int],
   value: Option[String],
+  valueType: Option[String],
```

Comment:
It could still have a type even if it's (conceptually) null, but that's not necessarily right...